doc/src/global-flow-conf.rst

```diff
@@ -6,10 +6,31 @@ Global (Site, User) Configuration Files
 Cylc site and user global configuration files contain settings that affect all
 suites. Some of these, such as the range of network ports used by cylc,
 should be set in the site ``flow.rc`` config file. Legal items,
-values, and system defaults are documented in (:ref:`SiteRCReference`).
+values, and system defaults are documented in :ref:`SiteRCReference`.
 
 Others, such as the preferred text editor for suite configurations,
-can be overridden by users in ``~/.cylc/flow/<CYLC_VERSION>/flow.rc``
+can be overridden by users in ``~/.cylc/flow/<CYLC_VERSION>/flow.rc``.
 
-.. literalinclude:: ../../cylc/flow/etc/flow.rc.eg
-   :language: none
+
+How to create a site or user flow.rc config file
+------------------------------------------------
+
+The ``cylc get-global-config`` command prints global config defaults,
+overridden by site global settings (if any), overridden by user global
+settings (if any).
+
+To generate a new global config file:
+
+1. Run ``$ cylc get-global-config > flow.rc``
+2. Edit any settings that you need to modify.
+3. Delete or comment out any seeting you do not need (to avoid inadvertently
+   overriding defaults or site settings that may change in the future).
+
+For all legal items, see the :ref:`SiteRCReference`.
```

Comment:
s/legal items/valid items?

Reply:
Sorry @kinow, I am not sure what this comment means. Can you please explain?

Reply:
Was just wondering if valid would sound better than legal? Although normally I use illegal item.

Reply:
Resolving as I think legal is fine

Reply:
Good thought. However I did not actually write this content, I copied it from the file that was previously included in raw format here (bar for making some formatting changes to sit right in RST format), i.e. the ``flow.rc.eg`` file, i.e. [this one](https://github.com/cylc/cylc-flow/blob/master/cylc/flow/etc/flow.rc.eg), so I wouldn't consider it in the scope of this PR to revise the text there.

I really just want to make sure the build warnings are addressed without losing any content that was useful & there before.

Reply:
(I'm the original author of the aforementioned line; valid might be better, but I doubt anyone will misunderstand legal).